tests/unit/components/saphyr/SaphyrEnzymeModal.spec.js

```diff
@@ -41,14 +41,15 @@ describe('SaphyrEnzymeModal.vue', () => {
       expect(wrapper.vm.selectedEnzymeId).toBe(1)
     })
 
-    it('has enzyme options', () => {
+    it('has enzyme options', async () => {
       let enzymeOptions = { enzymeOptions: [{ value: null, text: 'Please select an option' },
         { value: 1, text: 'enz1' },
         { value: 2, text: 'enz2' }]
       }
 
       wrapper.setData(enzymeOptions)
       expect(enzymeModal.enzymeOptions).toEqual(enzymeOptions.enzymeOptions)
+      await flushPromises()
```

Comment:
I think its needing this because on create it makes an async call to fetch the enzymeRequest, so maybe that can be mocked out in the mount and `await flushPromise` not needed

Reply:
We can't mock the method itself. Maybe we can mock the get? It is a deprecation warning at the moment so is not blocking the pull request.